<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KHC54QM" failures="2" tests="6" name="Test" time="0.143" errors="0" timestamp="2024-07-15T10:09:17 IST">
  <testcase classname="failed_testngsuite.class3" name="testsui5" time="0.024">
    <failure type="java.lang.AssertionError" message="both are diff expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: both are diff expected [2] but found [1]
at failed_testngsuite.class3.testsui5(class3.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testsui5 -->
  <testcase classname="failed_testngsuite.class3" name="testsui6" time="0.007"/>
  <testcase classname="failed_testngsuite.class2" name="testsui3" time="0.003"/>
  <testcase classname="failed_testngsuite.class2" name="testsui4" time="0.002">
    <failure type="java.lang.AssertionError" message="both names are diff expected [vina] but found [vin]">
      <![CDATA[java.lang.AssertionError: both names are diff expected [vina] but found [vin]
at failed_testngsuite.class2.testsui4(class2.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testsui4 -->
  <testcase classname="failed_testngsuite.class1" name="testsui1" time="0.003"/>
  <testcase classname="failed_testngsuite.class1" name="testsui2" time="0.002"/>
</testsuite> <!-- Test -->
